web-crawler/src/main/java/com/google/sps/webcrawler/WebCrawler.java

```diff
@@ -157,17 +159,17 @@ public String handleResponse(final HttpResponse response) throws IOException {
    */
   public Optional<NewsArticle> scrapeAndExtractHtml(String url) {
     try {
-      URL formattedUrlObj = new URL(url);
-      URL robotsUrlObj = new URL(formattedUrlObj.getProtocol(), formattedUrlObj.getHost(),
+      URL formattedUrl = new URL(url);
```

Comment:
What makes this a "formattedUrl"?  Maybe just pass in the URL object as the function param instead?